jekyll/_cci2/local-cli.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: classic-docs
+title: "The CircleCI Local Command Line Interface (CLI)"
+short-title: "Using the CircleCI Local CLI"
+description: "How to run local jobs with the CLI"
+categories: [troubleshooting]
+order: 10
+alias: /docs/2.0/local-jobs/
+---
+
+*[Build Environments]({{ site.baseurl }}/2.0/build/) > {{ page.title }}*
+
+* TOC
+{:toc}
+
+## Overview
+
+The `circleci` commands enable you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase. 
```

Comment:
"initial setup" seems like a redundant phrase. Isn't a setup already an initial action?

This sentence could also benefit from being split in two.

Reply:
Pre-existed.